packages/cms-cli/lib/prompts.js

```diff
@@ -0,0 +1,70 @@
+const API_KEY_REGEX = new RegExp(
+  '^([a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12})$',
+  'i'
+);
```

Comment:
I missed this before but this could be a regex literal.  Our prettier config is based on Airbnb's style guide which prefers literals over constructors when applicable.